sketches/M2_CAN_haz_bus/defines.h

```diff
@@ -35,26 +35,6 @@
 #define CMD_CAN_SENDRECV_ISOTP   0x48
 
 /* constants for setting baudrate for the CAN bus */
-uint32_t baud_rates_table[19] = {
```

Comment:
that doesn't immediately compute.  you're moving it out of a header file into a .cpp file.  that makes it *harder* to reference from other files.  am i missing something?